ruby-training/trainees/alan/test_katas/01/math_maddness/spec/math_maddness_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
+    
+    def self.add(first_number, second_number)
+       first_number +  second_number
+    end
+    
+    def self.substraction(first_number, second_number)
+        first_number - second_number
+    end
+
+    def self.multiply(first_number, second_number)
+        first_number * second_number
+    end
+
+    def self.divide(first_number, second_number)
+        first_number / second_number
+    end
+end
+describe Aritmetic_operations do
+    describe "#add" do
```

Comment:
You chose to use class methods (the ones starting with `self` and called directly from the class). The "#method_name" format (with an octothorpe "#") is for instance methods. The equivalent for class methods is the dot format (".method_name). Please change this for every describe block. 

To be clear, this should be changed to:

```ruby
describe ".add" do
  .
  .
  .
end
```